app/src/main/java/io/homeassistant/companion/android/onboarding/authentication/AuthenticationFragment.kt

```diff
@@ -87,6 +88,7 @@ class AuthenticationFragment : Fragment() {
                                     errorResponse: WebResourceResponse?
                                 ) {
                                     super.onReceivedHttpError(view, request, errorResponse)
+                                    Log.e(TAG, "onReceivedHttpError: $errorResponse")
```

Comment:
This will result in a fairly useless error message like:
```
2022-07-27 18:22:59.669 4100-4100/io.homeassistant.companion.android.debug E/AuthenticationFragment: onReceivedHttpError: android.webkit.WebResourceResponse@18c35d7
```
It should use the `getStatusCode()`and `getReasonPhrase()` functions to provide something usable.